ignite/pkg/jsonfile/jsonfile.go

```diff
@@ -0,0 +1,311 @@
+package jsonfile
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/buger/jsonparser"
+	"github.com/pkg/errors"
+
+	"github.com/ignite-hq/cli/ignite/pkg/tarball"
+)
+
+const (
+	keySeparator = "."
+)
+
+type (
```

Comment:
I find it not that readable when we combine all types into a single `type()` block.

Can we please separate them

Reply:
Why not? We can put all types/var/const statements in one place using this group of similar declarations. Most of the code styles encourage to use them, like the [uber code style](https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations)